ZAPD/ImageBackend.cpp

```diff
@@ -216,7 +216,7 @@ void ImageBackend::WritePng(const char* filename)
 
 void ImageBackend::WritePng(const fs::path& filename)
 {
-	WritePng(filename.c_str());
+	WritePng((char*)filename.c_str());
```

Comment:
Why is this cast needed? `.c_str()` should be returning a `const char*`, which is the expected type for that method, so I'm curious about this.

Reply:
To be honest I don't remember why that change was made. I would think it was a compiler issue but removing it works fine so your guess is as good as mine.